blog/migrations/0001_initial.py

```diff
@@ -34,7 +34,7 @@ class Migration(migrations.Migration):
             fields=[
                 ('id', models.AutoField(primary_key=True, serialize=False, verbose_name='ID', auto_created=True)),
                 ('sort_order', models.IntegerField(null=True, blank=True, editable=False)),
-                ('category', models.ForeignKey(related_name='+', to='blog.BlogCategory')),
+                ('category', models.ForeignKey(related_name='+', to='blog.BlogCategory', on_delete=models.CASCADE)),
```

Comment:
This is kind of strange, why was the 0001 initial migration changed?

Reply:
> The on_delete argument for ForeignKey and OneToOneField is now required in models and migrations. Consider squashing migrations so that you have fewer of them to update.

https://docs.djangoproject.com/en/2.0/releases/2.0/